docs/whirlpool/docs/03-Whirlpools SDK/02-Whirlpool Management/01-Create Pool.md

```diff
@@ -0,0 +1,94 @@
+3. **Initial Price**: Specify the initial price of token 1 in terms of token 2.
+4. **Funder**: This will be your wallet, which will fund the pool initialization.
+5. **Create instructions**: Use the `createConcentratedLiquidityPool()` function to create the pool.
+    ```tsx
+    import { createConcentratedLiquidityPool } from '@orca-so/whirlpools'
+
+    const { poolAddress, instructions, initializationCost } = await createConcentratedLiquidityPool(
+        Rpc,
+        tokenMintOne,
+        tokenMintTwo,
+        tickSpacing,
+        initialPrice,
+        wallet
+    );
+    ```
+6. **Submit Transaction**: Include the generated pool creation instructions in a Solana transaction and send it to the network using the Solana SDK.
+
+### Comparison
+| Feature            | Splash Pools       | Concentrated Liquidity Pools     |
+| ------------------ | ------------------ | -------------------------------- |
+| Complexity         | Low                | High                             |
+| Initial Parameters | Token mints, price | Token mints, tick spacing, price |
+| Capital Efficiency | Moderate           | High                             |
+| Ideal For          | Beginners          | Advanced Users                   |
+
+## 3. Usage Examples
+
+### Launching a Token Pair with a Splash Pool
+
+Suppose you want to launch a new memecoin and pair it with USDC. You can use `createSplashPoolInstructions()` to set up the pool quickly with a given initial price. This is ideal if you want to keep things simple and start earning trading fees with minimal configuration. For example, if a development team is building a launchpad for memecoins, Splash Pools are an ideal solution.
+
+### Creating a Concentrated Liquidity Pool for Efficiency
+
+If you want to maximize capital efficiency, use `createConcentratedLiquidityPool()` to define specific price ranges for your liquidity. This approach is beneficial when you expect price movements within certain bounds and want to concentrate liquidity accordingly. For example, a DeFi protocol might use a Concentrated Liquidity Pool to facilitate a stablecoin-stablecoin pair, where the price is expected to remain within a tight range. By concentrating liquidity in this range, the protocol can maximize returns for liquidity providers and reduce slippage for traders.
+
+## 4. Next Steps
+
+After creating a liquidity pool, the pool is still empty and requires liquidity for people to trade against. To make the pool functional, open a position and add liquidity at a ratio that matches the current price. This enables traders to swap between tokens and helps you start earning fees.
```

Comment:
When opening a position you don't have to worry about the ratio so probably good to not even mention that here. For CL pools your liquidity will need to be around the current price (otherwise it is still not tradable) but we can add that in the doc for opening a position